src/netcore/SnWebApplicationWithIdentityAndMemDb/SnWebApplicationWithIdentityAndMemDb/Data/ApplicationDbContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+
+namespace SnWebApplicationWithIdentity.Data
```

Comment:
In case of an in-mem implementation the whole `Data` folder can be removed because it contains the EF-related classes that we do not use.